deployment/ccip/view/v1_5/token_pool.go

```diff
@@ -0,0 +1,81 @@
+package v1_5
+
+import (
+	"encoding/hex"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+
+	"github.com/smartcontractkit/chainlink/deployment/common/view/types"
+)
+
+type TokenPoolContract interface {
+	Address() common.Address
+	Owner(opts *bind.CallOpts) (common.Address, error)
+	TypeAndVersion(*bind.CallOpts) (string, error)
+	GetToken(opts *bind.CallOpts) (common.Address, error)
+	GetSupportedChains(opts *bind.CallOpts) ([]uint64, error)
+	GetRemotePools(opts *bind.CallOpts, remoteChainSelector uint64) ([][]byte, error)
+	GetRemoteToken(opts *bind.CallOpts, remoteChainSelector uint64) ([]byte, error)
+}
+
+type RemoteChainConfig struct {
+	// RemoteTokenAddress is a hex representation of byte arrays
+	RemoteTokenAddress string
+	// RemotePoolAddresses are a hex representations of byte arrays
+	RemotePoolAddresses []string
+}
+
+type TokenPoolView struct {
+	types.ContractMetaData
+	Token              common.Address               `json:"token"`
+	RemoteChainConfigs map[uint64]RemoteChainConfig `json:"remoteChainConfigs"`
```

Comment:
"SupportedChains" aligns with convention of the contract

```suggestion
	SupportedChains map[uint64]RemoteChainConfig `json:"supportedChains"`
```

Reply:
No, SupportedChains is another methods+field on the contract
remoteChainConfigs doesn't have public method, but corresponds to s_remoteChainConfigs internal field